app/controllers/admins/members_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Admins
+  class MembersController < BaseController
+    def index
+      @members = Member.all
+    end
+
+    def show
+      @member = Member.find(params[:id])
+    end
+
+    def new
+      @member = Member.new
+    end
+
+    def create
+      @member = Member.new(member_params)
+      if @member.save
+        redirect_to action: 'index'
+      else
+        # TODO: add banner describing why form was invalid
+        redirect_to action: 'new'
```

Comment:
You should `render 'new'` so that you can use the `@member` object in the view. In the view, you can then grab the error messages from `@member.errors.full_messages` (a not so great example at https://github.com/calblueprint/roots-of-success/blob/master/app/views/application/_form_errors.html.slim)